polyfills/crypto/randomUUID/config.toml

```
@@ -0,0 +1,34 @@
+#### Aliases
+aliases = []
+
+#### Dependencies
+dependencies = []
+
+#### Specification Link
+# TODO update spec link when it settles
+spec = "https://wicg.github.io/uuid/"
+
+#### Documentation Link
+docs = "https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues"
+
+notes = [
+	"Uses progressively better ways to generate UUID's based on browser capabilities"
+]
+
+#### License
+license = "MIT"
+
+#### Browser compatibility
+[browsers]
+android = "18 - 91"
+bb = "*"
+chrome = "11 - 91"
+edge = "12 - 91"
+firefox = "26 - *"
+ios_saf = "6.1 - *"
+ie = "10 - *"
+ie_mob = "10 - *"
+opera = "15 - *"
+safari = "6.1 - *"
+firefox_mob = "26 - *"
```

```suggestion
firefox = "26 - 94"
ios_saf = "6.1 - *"
ie = "10 - *"
ie_mob = "10 - *"
opera = "15 - *"
safari = "6.1 - *"
firefox_mob = "26 - 94"
```

https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/95